src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ProjectReference.xaml

```diff
@@ -66,9 +66,17 @@
                   DisplayName="Exclude Assets"
                   Visible="True" />
 
-  <StringProperty Name="FullPath"
-                  Visible="False" />
-  
+  <StringProperty Name="ProjectFileFullPath"
```

Comment:
Nit: We should keep these files in alphabetical order for easier comparisons

Reply:
👍

Reply:
@davidwengier this is another candidate for the unit tests.